src/commands/local.ts

```diff
@@ -799,6 +811,24 @@ async function startServerHttp(
     });
 }
 
+function getProjectFunctions(
+    port: number,
+    projectConfiguration: ProjectConfiguration,
+): DeployCodeFunctionResponse[] {
+    const functions: DeployCodeFunctionResponse[] = [];
+
+    if (projectConfiguration.functions.length > 0) {
+        projectConfiguration.functions.forEach((f) => {
+            functions.push({
+                cloudUrl: `http://localhost:${port}/.functions/${f.name}`,
+                id: f.name,
+                name: f.name,
+            });
+        });
+    }
+    return functions;
```

Comment:
I would simplify this code like this:

```suggestion
    return projectConfiguration.functions.map((f) => ({
                cloudUrl: `http://localhost:${port}/.functions/${f.name}`,
                id: f.name,
                name: f.name,
            }));
```